tools/cell_census_builder/consolidate.py

```diff
@@ -5,6 +5,7 @@
 
 import tiledbsoma as soma
 
+from .globals import SOMA_TileDB_Context
 from .mp import create_process_pool_executor
 
 if soma.get_storage_engine() == "tiledb":
```

Comment:
this import of `tiledb` should be moved into the `consolidate_tiledb_object` function (and yeah, this is my latent bug :-)).

Reply:
fixed!